FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -57,5 +58,23 @@ public void ExecuteSetEngineTest(string commandText)
             var value = conn.CreateCursor().Execute(commandText);
             Assert.IsNotEmpty(value.Data);
         }
+
+        [Test]
+        public void ExecuteTestInvalidCredentials()
+        {
+            var connString = $"database={Database};username={Username};password=wrongPassword;endpoint={Endpoint};";
+
+            using var conn = new FireboltConnection(connString);
+
+            try
```

Comment:
Can we use `Assert.Throws` and `Assert.ThrowsAsync` in tests, when we want to check that a specific error is thrown? I believe we can fetch exception object to check the message as well.
https://docs.nunit.org/articles/nunit/writing-tests/assertions/classic-assertions/Assert.Throws.html